### ISSUE TITLE
Overlapping shadows in monocle layout

### ISSUE THREAD

**Issue by karasuma-chitose** [baskerville/bspwm#945]
Here's a silly question about a minor annoyance.
When in monocle mode (assuming you're using gaps and compton), every time you open a new window, you get a new shadow on top of the previous shadows which starts looking rather unsightly.
Would it be possible to have compton somehow display only one shadow for windows stacked in monocle layout?

**Comment by astier** [comment]
Related #727

**Comment by jallbrit** [comment]
Just like @astier said, this is basically a duplicate of #727. Please close the issue @karasuma-chitose.

**Comment by emanuele6** [comment]
I wouldn't say they are duplicates!

* This issue is reporting a problem;
* #727 is proposing to change the implementation of the `monocle` layout (which would prevent this problem).

**Comment by jallbrit** [comment]
@emanuele6 I agree that they are related and not *exact* duplicates; however, it would appear that they are the same root issue: monocle mode stacks nodes and doesn't hide the nodes underneath. Technically, this issue only asks to hide the *shadows* generated by the stacked/hidden nodes, while #727 wants to hide the nodes entirely which IMO is the root issue here.

In my opinion it would make more sense to keep all discussion limited to #727.
